app/helpers/mailing_lists_helper.rb

```diff
@@ -52,17 +52,25 @@ def format_mailchimp_sync(mailing_list)
 
   private
 
+  # FIXME - appears redundant, subscriptions are also managed via Person::SubscriptionsController
+  # subscribed? returns true (ie. subscribed via role) but without specific person subscription
+  # Additionally, creating person subscription via list enforces abilities on list wheres creating via person enforces abilities on person
```

Comment:
In 2d191397443e68180611b327d6562ee7f1c06754 wurde der `group_mailing_list_subscription_path` anstelle des `group_mailing_list_user_path` gesetzt. Der zugehörige Subscriber::UserController wurde bereits in 1f13c45f6e19852cb04ca41d17aefefb47b32800 gelöscht und durch den Person::SubscriptionsController ersetzt. Wahrscheinlich ging bereits dann diese Methode hier vergessen und wurde danach nicht korrekt gefixt. IMHO sollte hier der Person::SubscriptionsController aufgerufen werden, dann braucht es auch das `if` für die subscription nicht mehr. In `mailing_lists/_actions_show.html.haml` sollte `if MailingList.subscribable.include?(entry)` ersetzt werden durch `if entry.subscribable?`. 
Pass dies doch bitte noch an.